src/sentry/db/models/manager/base_query_set.py

```diff
@@ -4,12 +4,42 @@
 from django.db import router
 from django.db.models import QuerySet
 
+from sentry.signals import post_update
+
 
 class BaseQuerySet(QuerySet, abc.ABC):
-    # XXX(dcramer): we prefer values_list, but we can't disable values as Django uses it
-    # internally
-    # def values(self, *args, **kwargs):
-    #     raise NotImplementedError('Use ``values_list`` instead [performance].')
+    def __init__(self, *args, **kwargs):
```

Comment:
@wedamija should we modify the base query set or should we make one just for the group manager?

Reply:
I initially built this into the group manager, but it ended up needing a bunch of if statements in various places. This feels more generic, and doesn't change how the base queryset works by default.